src/server/game/Server/WorldSession.cpp

```diff
@@ -1665,9 +1665,10 @@ bool WorldSession::IsRightUnitBeingMoved(ObjectGuid guid)
     GameClient* client = GetGameClient();
 
     // the client is attempting to tamper movement data
+    // edit: this wouldn't happen in retail but it does in TC, even with a legitimate client.
```

Comment:
the comment is probably wrong but i dont want to go and test it today (all you need is to find some area restricted vehicle and go out of bounds with it)

Reply:
I'm not sure myself. In the next weeks, I'll dig through the sniffs I have and try to confirm/infirm.

Reply:
> I'm not sure myself. In the next weeks, I'll dig through the sniffs I have and try to confirm/infirm.

jump irc i think we have some restricted area vehicles sniffs.